ex2.js

```diff
@@ -0,0 +1,17 @@
+/*Scrieți o funcție pentru a genera un număr aleatoriu mai mare sau egal 
+cu o variabilă start și strict mai mic decât o variabilă end . 
+Aceste două variabile sunt capetele intervalului pentru numărul general: [start, end)
+*/
+
+const getRandomNumber = (start, end) => {
+    return Math.random () * (end - start ) + start;
+}
+console.log(getRandomNumber(3, 5)); // Nu ne va da un numar intreg 
```

Comment:
```suggestion
// Nu ne va da un numar intreg 
console.log(getRandomNumber(3, 5));
```
(opt) comentariile se pun inaintea codului, pe o line aparte